pkg/util/util.go

```diff
@@ -72,6 +72,24 @@ func NewGCPSecretCR(namespace, creds string) *corev1.Secret {
 	}
 }
 
+// NewGCPSecretCRV2 returns a Secret CR formatted for GCP for use in projectreference controller until refactor
```

Comment:
I don't fully understand until when this function will be used and which refactoring is needed here, can you detail that out a bit?

Reply:
Since clusterdeployment controller is still in place we need to keep the original function `NewGCPSecretCR` intact since I do not want to modify that code, Since in the new code the ProjectClaim specifies the Namespaces and Name of the secret to be created I added namespacedName which contains the name and namespace to a `v2` version of this function. Once we remove the clusterdeployment controller, we will be able to delete the original function and rename this one.

Reply:
Understood. Can you put that to the comment? :D

Reply:
See https://github.com/rafael-azevedo/gcp-project-operator-1/pull/1